app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::OrdersController < ApiController
+  before_action :authenticate_api_v1_user!
+
+  def create
+    create_order
+    begin
+      dish = Dish.find(params[:dish_id])
+    rescue ActiveRecord::RecordNotFound
+      render json: { message: "Something went wrong, you did not add a dish to your order" }, status: :unprocessable_entity and return
+    end
+    @order.add(dish, dish.price)
+    render json: {message: "Successfully added Pizza"}, status: :ok
+  end
+
+  private
+
+  def create_order
+    user = current_api_v1_user
+    @order = Order.create(user: user)
+  end
```

Comment:
In your regular namespace you are checking for an existing order by checking the cookie, right? I think you need an mechanism to check it the user already has an order open. otherwise you end up with several orders.... you know what I mean, we've covered this when building the app.